docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -583,18 +601,21 @@ If you are at this stage, you must use following util packages from CAPI v1.11 (
 import (
 	...
     "sigs.k8s.io/cluster-api/util/conditions"
-    v1beta1conditions "sigs.k8s.io/cluster-api/util/conditions/deprecated/v1beta1"
+    deprecatedv1beta1conditions "sigs.k8s.io/cluster-api/util/conditions/deprecated/v1beta1"
     "sigs.k8s.io/cluster-api/util/patch"
 )
 ```
 
 - the `conditions` package provides access to utils for managing `metav1.Conditions` in `status.conditions`
-- the `v1beta1conditions` package alias provides access to utils for managing `clusterv1beta2.Conditions` in `status.deprecated.v1beta1.conditions`
+- the `deprecatedv1beta1conditions` package alias provides access to utils for managing `clusterv1beta2.Conditions` in `status.deprecated.v1beta1.conditions`
 - the `patch` package provides access to utils for patching objects in this phase
 
 Important!
-The package `sigs.k8s.io/cluster-api/util/conditions/deprecated/v1beta1`" is going to be removed from CAPI when the
-v1beta1 removal will happen (tentative Aug 2026).
+- Please pay special attention to use `sigs.k8s.io/cluster-api/util/patch` import everywhere,
+  because if by mistake you use `sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch` in some files this won't lead to compile errors
+  but conditions are going to be dropped at runtime.
```

Comment:
```suggestion
  because using `sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch` by mistake could in some cases lead to dropping conditions at runtime,
  while not having compile errors.
```